Covid-19 _Real-time_Notification/Covid.py

```diff
@@ -0,0 +1,38 @@
+from plyer import notification
+import requests
+from bs4 import BeautifulSoup
+import time
+
+def notify_user(title, message):
+    notification.notify(
+    title = title,
+    message = message,
+    app_icon = "./Covid-19_Real-time_Notification/Notify_icon.ico" ,
+    timeout = 3)
+    
+def getInfo(url):
+    r = requests.get(url)
+    return r.text
+
+
+if __name__ == '__main__':
+    while True:
+        HtmlData = getInfo('https://www.mohfw.gov.in/')
+        soup = BeautifulSoup(HtmlData, 'html.parser')
+    
+        myData=""
+        for tr in soup.find('table').find('tbody')[1].find_all('tr'):
+            myData += tr.get_text()
+        
+        myData = myData[1:]
+        itemList = myData.split("\n\n")
+    
+        states = list(map(str,input("Enter the name of state:"))
```

Comment:
This is the error (returned by bot also): 

```suggestion
        states = list(map(str,input("Enter the name of state:")))
```

Also this is not logically correct, every time after the sleep time is over, the script will ask for the states. I think providing the states only once is better option. Shift this statement before while loop.